gateway/src/test/java/io/zeebe/gateway/api/util/GatewayTest.java

```diff
@@ -42,4 +43,9 @@ public void setUp() {
     client = gatewayRule.getClient();
     brokerClient = gatewayRule.getBrokerClient();
   }
+
+  @After
+  public void tearDown() {
+    // nothing to do
```

Comment:
What does this do?

Reply:
Allows any GatewayTest to just overwrite it, but just does not nothing normally. I added it for consistency with the `@Before`